src/Module.zig

```diff
@@ -3408,7 +3428,8 @@ pub fn getErrorValue(mod: *Module, name: []const u8) !std.StringHashMapUnmanaged
     errdefer mod.global_error_set.removeAssertDiscard(name);
 
     gop.entry.key = try mod.gpa.dupe(u8, name);
```

Comment:
In order to not leak on allocation failure, this resource needs an errdefer, or better yet, `inv_error_map.ensureCapacity` before it, and then `inv_error_map.appendAssumeCapacity` below.